### ISSUE TITLE
Find alternative to map sitting above list

### ISSUE THREAD

**Issue by rafelafrance** [MapofLife/datasets#35]
From 'both' or 'everything' view

https://3.basecamp.com/3232053/buckets/1126873/todos/179805387

**Comment by rafelafrance** [comment]
Have we settled on a design for this?

**Comment by funkycoda** [comment]
Let me ponder this later today or over the weekend. Let's hold of on this till Monday

**Comment by jmalczyk** [comment]
No, we have not. Given the width of the table I'm not sure there is one.
You could put the map  and lists in a bootstrap column so it sits in the
middle or on the right on really wide screens, and then on top on narrow
ones.  Worth a try, might behave weird though.

**Comment by jmalczyk** [comment]
Pushing this to next iteration.
